cypress/support/a11y/checks/has-page-title.js

```diff
@@ -0,0 +1,29 @@
+const DEFAULT_TITLE = "MonComptePro";
```

Comment:
This is just a copy/paste from the title set in the ejs layout, I guess we should put it somewhere else to prevent syncing issues.